pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -1071,8 +1071,15 @@ func TestPodTimestamp(t *testing.T) {
 			queue.MoveAllToActiveQueue()
 		}
 	}
+	backoffPod := func(pInfo *podInfo) operation {
+		return func() {
+			queue.backoffPod(pInfo.pod)
+		}
+	}
 	flushBackoffQ := func() operation {
 		return func() {
+			// wait for pod to complete backoff.
+			time.Sleep(time.Second)
```

Comment:
Can you collaborate why this is needed? And it seems to be able to transfer to a more grace solution like channel or wait?

Reply:
Sure. Thanks. I transfer to a more grace solution. Done